pkg/git/git.go

```diff
@@ -43,6 +43,15 @@ func Commit(message string) error {
 	return err
 }
 
+func CommitAmend() error {
+	log.Infof("Amending the changes...")
+	err := gitExec("commit", "--amend", "--no-edit")
+	if err == nil {
+		log.Info("Amended the change to the previous commit.")
+	}
+	return err
+}
+
```

Comment:
`CommitAmend()` seems unused. Please remove or show where it's used.

Another story is whether we want to ever amend commits in the first place.

Reply:
removed